src/Run.py

```diff
@@ -51,56 +35,27 @@ def main():
     print("=========================")
     print("Training..")
 
-    # print("\nStupid Classifier has been training..")
-    #myStupidClassifier.train()
-    # print("Done..")
-
-    # print("\nPerceptron has been training..")
-    #myPerceptronClassifier.train()
-    # print("Done..")
-    
-    # print("\nLogistic Regression has been training..")
-    #myLRClassifier.train()
-    # print("Done..")
-
     print("\nMLP has been training..")
     myMLP.train()
     print("Done..")
 
     # Do the recognizer
     # Explicitly specify the test set to be evaluated
-   # stupidPred = myStupidClassifier.evaluate()
-   # perceptronPred = myPerceptronClassifier.evaluate()
-   # lrPred = myLRClassifier.evaluate()
     MLPPred = myMLP.evaluate()
 
     
     # Report the result
     print("=========================")
     evaluator = Evaluator()
 
-    # print("Result of the stupid recognizer:")
-    #evaluator.printComparison(data.testSet, stupidPred)
-  #  evaluator.printAccuracy(data.testSet, stupidPred)
-
-    # print("\nResult of the Perceptron recognizer:")
-    #evaluator.printComparison(data.testSet, perceptronPred)
-   # evaluator.printAccuracy(data.testSet, perceptronPred)
-    
-    # print("\nResult of the Logistic Regression recognizer:")
-    #evaluator.printComparison(data.testSet, lrPred)    
-  #  evaluator.printAccuracy(data.testSet, lrPred)
-
     print("Result of the mlp recognizer:")
-    evaluator.printComparison(data.testSet, MLPPred)
+    #evaluator.printComparison(data.testSet, MLPPred)
```

Comment:
Brauchen wir diesen Kommentar?

Reply:
Nicht unbedingt. Der ist nur blöd weil man 1000 extra Zeilen in Output hat.

Reply:
Meinte, ob wir nicht die Zeile löschen sollten oder einen Parameter hinzufügen sollten.

Reply:
Achso. Vielleicht Zeile einfach raus, keiner will genau sehen welche Beispiele falsch waren.